dojo/settings/settings.dist.py

```diff
@@ -841,6 +842,11 @@ def generate_url(scheme, double_slashes, user, password, host, port, path):
 # See http://docs.djangoproject.com/en/dev/topics/logging for
 # more details on how to customize your logging configuration.
 LOGGING_HANDLER = env('DD_LOGGING_HANDLER')
+
+LOG_LEVEL = env('DD_LOG_LEVEL')
+if not LOG_LEVEL:
+    LOG_LEVEL = 'DEBUG' if DEBUG else 'INFO'
```

Comment:
Why not pass `INFO` as default value to `DD_LOG_LEVEL`?

Reply:
Because I want it to be aligned with DD_DEBUG by default so we don't have to add DD_LOG_LEVEL everywhere